### PR TITLE
Added LTE property collection.

### PR DESCRIPTION
**Pull request by aoifeboyle**
*Don't merge yet*

I expanded the stub for a property collection for the LTE Plasma but I'm not sure what I've done is the correct / ideal way to do it. Please look through and make suggestions if necessary.

### CODE REVIEW

File: tardis/plasma/properties/property_collections.py [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
+    LevelPopulationLTE, LevelNumberDensity, PhiSahaLTE, GElectron,
+    IonizationData, NumberDensity, IonNumberDensity, LinesLowerLevelIndex,
+    LinesUpperLevelIndex, TauSobolev, TRadiative, AtomicData, Abundance,
+    Density, TimeExplosion)
 
 class LTEInputs(list):
-    pass
-
-
+    def __init__(self):
```
Comment (wkerzendorf) [inline tardis/plasma/properties/property_collections.py]: I inherit from list so you can just do `self.append`. But other than that it looks good. you sort of see the concept of what I want, right?

Comment (aoifeboyle) [inline tardis/plasma/properties/property_collections.py]: @wkerzendorf Yeah, I think the property groups make a lot of sense! It makes things a lot tidier anyway.

**Comment by aoifeboyle** [comment]
@wkerzendorf Do you know why the coverage test failed here?

File: tardis/plasma/standard_plasmas.py [hunk 2]
```diff
@@ -1,26 +1,14 @@
...
 
     def __init__(self, t_rad, abundance, density, time_explosion, atomic_data):
-        plasma_modules = [TRadiative, BetaRadiation, LevelBoltzmannFactor,
-                          Levels, Lines, AtomicData, Abundance, SelectedAtoms,
-                          AtomicMass, LTEPartitionFunction, LevelPopulationLTE,
-                          PhiSahaLTE, GElectron, IonizationData, Density,
-                          NumberDensity, IonNumberDensity, LevelNumberDensity,
-                          LinesLowerLevelIndex, LinesUpperLevelIndex,
-                          TauSobolev, TimeExplosion]
+        plasma_modules = LTEInputs().list
```
Comment (kaushik94) [inline tardis/plasma/standard_plasmas.py]: @aoifeboyle I think the test are failing due to this. I'm not sure but I guess it should be `list(LTEInputs())`

Comment (wkerzendorf) [inline tardis/plasma/standard_plasmas.py]: I think this is allright. But I would use it as a list. I think that `LTEInputs` should also only contain the inputs not every lte plasma module.

**Comment by aoifeboyle** [comment]
@wkerzendorf This seems to be working well now - are you happy with it?

File: tardis/plasma/properties/property_collections.py [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+    LevelPopulationLTE, LevelNumberDensity, PhiSahaLTE, GElectron,
+    IonizationData, NumberDensity, IonNumberDensity, LinesLowerLevelIndex,
+    LinesUpperLevelIndex, TauSobolev, TRadiative, AtomicData, Abundance,
+    Density, TimeExplosion)
 
-class LTEInputs(list):
-    pass
-
-
+class LTEProperties(list):
```
Comment (wkerzendorf) [inline tardis/plasma/properties/property_collections.py]: Okay the ideas for these collections is more that you add together multiple collections. So here's my suggestion of how to do this:

```python

class PlasmaPropertyCollection(list):
    pass

generic_inputs = PlasmaPropertyCollection([AtomicData, TRadiative, ...])
lte_processing = PlasmaPropertyCollection([LevelPopulationLTE, ...])


my_lte_properties = generic_inputs + lte_processing + whatever else you need
```

**Comment by wkerzendorf** [comment]
@aoifeboyle let me know when you want to merge what

**Comment by aoifeboyle** [comment]
I'm happy to merge both PRs now if you're happy with them. I have one more test to add later today and then I'm going to get stuck into the legacy plasma stuff. @wkerzendorf

**Pull request merged**